ui/src/index.css

```diff
@@ -16,6 +15,20 @@
   -webkit-text-size-adjust: 100%;
 }
 
+html, body {
+  /* ensure document root and body have no height, padding, or margin impact on react content */
+  height: 100%;
+  box-sizing: border-box;
```

Comment:
`border-box` means include any padding and margin in the calculation of dimensions (height etc). Not strictly necessary here since we are setting 0 padding and margin, but nothing lost by including it regardless.